src/api/l_data_soundData.c

```diff
@@ -9,8 +9,32 @@ static int l_lovrSoundDataGetBlob(lua_State* L) {
   return 1;
 }
 
+static int l_lovrSoundDataAppend(lua_State* L) {
+  SoundData* soundData = luax_checktype(L, 1, SoundData);
+  Blob* blob = luax_totype(L, 2, Blob);
+  SoundData* sound = luax_totype(L, 2, SoundData);
+  lovrAssert(blob || sound, "Invalid blob appended");
```

Comment:
Could use `luaL_argerror` here to get a conventional "Bad argument N ..." error message.  It could even get added as an else down below so an extra if isn't needed here.

Reply:
that's much nicer! gret call :)